backend/resolver/aws/aws.go

```diff
@@ -136,3 +150,48 @@ func (r *res) Search(ctx context.Context, typeURL, query string, limit uint32) (
 		return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("cannot search for type '%s'", typeURL))
 	}
 }
+
+func (r *res) AutoComplete(ctx context.Context, typeURL, search string, limit uint64) ([]*resolverv1.AutocompleteResponse_AutocompleteResult, error) {
+	if r.topology == nil {
+		return nil, fmt.Errorf("to use the autocomplete api you must first setup the topology service")
+	}
+
+	// TODO (mcutalo): Before implementing another resource to support autocomplete
+	// consider abstracting this into the topology service.
+	searchRequest := &topologyv1.SearchRequest{
+		PageToken: "0",
+		Sort: &topologyv1.SearchRequest_Sort{
+			Direction: topologyv1.SearchRequest_Sort_ASCENDING,
+			Field:     "column.id",
+		},
+		Filter: &topologyv1.SearchRequest_Filter{
+			TypeUrl: typeURL,
+			Search: &topologyv1.SearchRequest_Filter_Search{
+				Field: "column.id",
+				Text:  search,
+			},
+		},
+	}
+
+	// Limit is optional, if one is not set we use the default set by the topology search api
+	if limit > 0 {
+		searchRequest.Limit = limit
```

Comment:
you probably want your own default limit that's lower than whatever the default limit of the topo API is

Reply:
Yeah true let me do so